csharp/Ice/Greeter/Client/Program.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ZeroC, Inc.
+
+// Slice module VisitorCenter in Greeter.ice maps to C# namespace VisitorCenter.
+using VisitorCenter;
+
+// Create an Ice communicator to initialize the Ice runtime. The communicator is disposed before the program exits.
+using Ice.Communicator communicator = Ice.Util.initialize(ref args);
+
+// GreeterPrx is a class generated by the Slice compiler. We create a proxy from a communicator and a "stringified
+// proxy" with the address of the target object.
+// If you run the server on a different computer, replace localhost in the string below with the server's hostname
+// or IP address.
+GreeterPrx greeter = GreeterPrxHelper.createProxy(communicator, "greeter:tcp -h localhost -p 4061");
+
+// Send a request to the remote object and get the response.
+// TODO: remap to GreetAsync with cs:identifier
+string greeting = await greeter.greetAsync(Environment.UserName);
```

Comment:
I think we should only demo the client-side async API. The sync API is for backwards compatibility - you probably shouldn't use it in new applications.